model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -704,13 +705,31 @@ def _create_default_perf_analyzer_config(
             model_config.get_field("name"), self._config
         )
 
-        perf_config_params = {"batch-size": 1, "concurrency-range": 1}
+        default_concurrency = self._calculate_default_concurrency(model_config)
+
+        perf_config_params = {
+            "batch-size": DEFAULT_BATCH_SIZES,
+            "concurrency-range": default_concurrency,
+        }
         default_perf_analyzer_config.update_config(perf_config_params)
 
         default_perf_analyzer_config.update_config(model.perf_analyzer_flags())
 
         return default_perf_analyzer_config
 
+    def _calculate_default_concurrency(self, model_config: ModelConfig) -> int:
+        default_max_batch_size = model_config.max_batch_size()
+
+        # string format is: "<count>:GPU"
```

Comment:
I'm not very confident in this. Is this always the format? What about CPU case?  

I don't like that this code needs to know a string format and to parse it. Is there a way to push this support into the model_config class?

Reply:
Agreed. Thinking more about this, the code doesn't work for the case where we have multiple different GPUs and/or a mix of KIND_CPU + KIND_GPU. I'll create a new method in `model_config` and add some unit testing.